api/v1alpha1/nonadminbackupstoragelocation_types.go

```diff
@@ -71,6 +71,10 @@ type NonAdminBackupStorageLocationStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=nonadminbackupstoragelocations,shortName=nabsl
+// +kubebuilder:printcolumn:name="Approved",type="string",JSONPath=".status.conditions[?(@.type=='ClusterAdminApproved')].status"
+// +kubebuilder:printcolumn:name="BSL-Status",type="string",JSONPath=".status.veleroBackupStorageLocation.status.phase"
```

Comment:
from velero
```console
$ velero backup-location get
NAME              PROVIDER   BUCKET/PREFIX           PHASE         LAST VALIDATED                  ACCESS MODE   DEFAULT
velero-sample-1   aws        my-bucket-name/velero   Unavailable   2025-03-06 12:54:30 +0000 UTC   ReadWrite     true

$ velero backup get
NAME     STATUS   ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
backup   Failed   0        0          2025-03-06 12:56:14 +0000 UTC   29d       velero-sample-1    <none>
```

would change `BSL-Status` to `Storage Location Phase` (if spaces are allowed) and `Status` to `Phase`

Reply:
does the order matter?

currently is APPROVED ->  BSL-Phase -> Phase (NABSL)

would be better to change to NABSL phase -> Approved -> Storage Location Phase?

Reply:
I would really not like to have spaces as this isn't a good CLI/Shell practice in my opinion. It's hard to use awk/sed/cut for those as well they make have affect on the alignment in various terminal systems.